suites/pacific/rbd/tier-3_journal_mirroring_regression.yaml

```diff
@@ -0,0 +1,203 @@
+#===============================================================================================
```

Comment:
why it need a separate test suite?

Reply:
because i wanted to maintain separate test suites for separate features like we started doing sometime back.